modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -941,12 +941,10 @@ impl ServerModule for Wallet {
                 async |_module: &Wallet, context, _params: ()| -> () {
                     check_auth(context)?;
 
+                    // api_endpoint! calls dbtx.commit_tx_result
                     let mut dbtx = context.dbtx();
-
                     dbtx.insert_entry(&ConsensusVersionVotingActivationKey, &()).await;
 
-                    dbtx.commit_tx_result().await.map_err(|e| ApiError::server_error(e.to_string()))?;
```

Comment:
@m1sterc001guy

Reply:
IMO since the api endpoint macro calls `commit_txt`, the `ApiEndpointContext` should return a non-committable DB transaction instead of the committable one. Will look into making this change.

Reply:
https://github.com/fedimint/fedimint/pull/6875